slycot/tests/test.py

```diff
@@ -64,6 +64,111 @@ def test_td04ad_static(self):
             nr,a,b,c,d = transform.td04ad(rc,nin,nout,index,den,num)
 
 
+# ===================================================
+# Begin tb05ad tests
+import numpy as np
+
+from numpy.testing import assert_raises, assert_almost_equal
+from numpy.testing import assert_array_almost_equal, dec
+
+CASES = {}
+
+CASES['fail1'] = {'A': np.array([[-0.5,  0.,  0.,  0. ],
+                              [ 0., -1.,  0. ,  0. ],
+                              [ 1.,  0., -0.5,  0. ],
+                              [ 0.,  1.,  0., -1. ]]),
+                   'B': np.array([[ 1.,  0.],
+                             [ 0.,  1.],
+                             [ 0.,  0.],
+                             [ 0.,  0.]]),
+                   'C': np.array([[ 0.,  1.,  1.,  0.],
+                             [ 0.,  1.,  0.,  1.],
+                             [ 0.,  1.,  1.,  1.]])}
+
+n = 20
+p = 10
+m = 14
+np.random.seed(40)
```

Comment:
I think I forgot to click a button somewhere around here when doing the review.

I also think this test case (class? whatever) should be in it's own file (`test_tb05ad.py`).